src/cls/IPM/Utils/Module.cls

```diff
@@ -2245,7 +2245,16 @@ ClassMethod GetFileLines(pFileName As %String, Output pOutput) As %Status [ Inte
 /// 	<li>{$binDir} - the instance's bin directory.</li>
 /// 	<li>{$libDir} - the instance's lib directory.</li>
 /// 	<li>{$webroot} - the instance's constructed url with host and port (e.g. http://123.45.678.90:52773/) </li>
-/// 	<li>{$dbrole} - the current namespace's database role</li>
+/// 	<li>{$dbrole} - (deprecated in favor of {$globalsDbRole}) the current namespace's globals database role</li>
+/// 	<li>{$globalsDbRole} - the current namespace's globals database role, functionally equivalent to {$dbrole} in legacy packages.</li>
+/// 	<li>{$codeDbReadRole} - [In WebApplication only] the current namespace's routine database role (will be created if needed)</li>
```

Comment:
I know it has side effects but still worth moving the role creation here. I don't think it should be for web apps only since a customer may use it for the web app and then want to use it in an invoke as well